razzle.config.js

```diff
@@ -42,6 +42,17 @@ module.exports = {
             // externals means files not generated by webpack
             // an alternative would be html-webpack-plugin
             caches: 'all',
+            externals: [
+              '/public/manifest.json',
+              '/public/images/icons/icon-72x72.png',
```

Comment:
do we need these images here?

Reply:
I think these only need to be here if the application makes a network request for them. I do not think they need to be here if they are here for PWA, manifest, reasons.